src/api/RequestApi.ts

```diff
@@ -29,8 +28,8 @@ const transformInRequestFromSP = (request: IResponseItem): IInRequest => {
     MPCN: request.MPCN,
     SAR: request.SAR,
     workLocation: request.workLocation,
-    isNewCivMil: request.isNewCivMil,
-    isTraveler: request.isTraveler,
+    isNewCivMil: request.isNewCivMil ? "yes" : "no",
+    isTraveler: request.isTraveler ? "yes" : "no",
```

Comment:
Can we discuss why you changed everything to yes/no instead of leaving it as a boolean?